### ISSUE TITLE
Feedback for 0.08.4

### ISSUE THREAD

**Issue by noahmaranesi** [tyirvine/Informant#2]
Hi!

Great app! Here's my feedback (more so suggestions 😅):

- Allow `⌘I` to be used. The current error says "This keyboard shortcut cannot be used as it's already used by the “Italic” menu item." when trying to use `⌘I`. It's also used by the Get Info menu item, not Italic (in Finder).
- It would be nice for users to be prompt the Accessibility.prefpane, either automatically, or by some button on that screen. Here is an Apple Script you can use:

    ```AppleScript
    tell application "System Preferences"
        tell pane id "com.apple.preference.security" to reveal anchor "Privacy_Accessibility"
        activate
    end tell
    ```

- An option to *completely* hide the menu bar icon. Activate application by opening the application (Finder, Spotlight) and use it purely in keyboard-shortcut-mode. You need to also change the icon so it's not the same as the Fast Switch menu icon!
- It would be great for the window to re-popup (?) when switching Desktops/Spaces too!

Thanks for the great app!

**Comment by tyirvine** [comment]
Thank you for the feedback! I'm not sure how to approach the shortcut as that's being handled by a package atm. But I could try and find a workaround.

I'll definitely be adding the rest though as it's highly requested.

I just am trying to set the project up for open source development and have hit some snags so I haven't got around to this yet.

**Comment by tyirvine** [comment]
Points 2 and 3 were addressed and added in 0.08.6! 🎉

I'm not going to pursue point 1 at this time
